src/main/webapp/encounters/encounterMediaGallery.jsp

```diff
@@ -1077,7 +1079,7 @@ function enhancerDisplayFeature(el, opt, focusAnnId, feat, zdelta) {
     var scale = el.data('enhancerScale') || 1;
 console.log('FEAT!!!!!!!!!!!!!!! scale=%o feat=%o', scale, feat);
     var focused = (feat.annotationId == focusAnnId);
-    var fel = $('<div title="Annot" style="z-index: ' + (31 + (zdelta||0)) + ';" class="image-enhancer-feature" />');
+    var fel = $('<div data-encid="' + feat.encounterId + '" data-idx="' + zdelta + '" title="Annot" style="z-index: ' + (31 + (zdelta||0)) + ';" class="image-enhancer-feature" />');
```

Comment:
just wondering if this `data-idx` is used again somewhere else?  cuz i couldnt find it -- wondering if i missed it.